README.md

```diff
@@ -14,6 +14,12 @@ We have two key phrases that we use throughout the app:
 1. **Competing Entity**: This is used for any competitor (individual or teams) in a show
 2. **Contestant**: This is used for any competitor in a league
 
+This app requires setting a local environment variable. Here's how to set up this variable: 
+1. Generate a local secret key. One way to do this is to run `openssl rand -base64 32`
```

Comment:
nit: these should all be `1.` so that they never need to be renumbered.

```suggestion
1. Generate a local secret key. One way to do this is to run `openssl rand -base64 32`
1. In your `.env.development.local`, create a variable named `SESSION_SECRET`
1. Set this variable equal to the value you generated in step 1.
1. Stop and start your app.

```